### PR TITLE
FE-1371: fix etherscan validation logic

### PR DESCRIPTION
**Pull request by tremarkley**
The etherscan validation logic is currently broken because it expects a `qResult` field in the response. According to the [etherscan api docs](https://docs.etherscan.io/api-endpoints/contracts#get-contract-source-code-for-verified-contract-source-codes) the response returns a `result` field. This PR switches the validation to use the `result` field.

Fixes #[FE-1371](https://linear.app/optimism/issue/FE-1371)

### CODE REVIEW

**Comment by tremarkley** [comment]
Current dependencies on/for this PR:
* master
  * **PR #299** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/299" target="_blank">Graphite</a>  👈
    * **PR #300** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/300" target="_blank">Graphite</a> 
      * **PR #301** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/301" target="_blank">Graphite</a> 
        * **PR #302** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/302" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/299?utm_source=stack-comment).

File: src/validate.ts [hunk 1]
```diff
@@ -220,22 +220,21 @@ export const validate = async (
           // Make sure the token is verified on Etherscan.
           // Etherscan API is heavily rate limited, so sleep for 1s to avoid errors.
           await sleep(1000)
-          const { qResult } = await (
+          const { result: etherscanResult } = await (
```
Comment (roninjin10) [inline src/validate.ts:223]: places like here we should be .catch(e => {
  console.error('unble to fetch etherscan', e, {chain, urlSarchParams}
}

Making it maximally easy to debug

**Review by roninjin10 — APPROVED** [review]

**Comment by tremarkley** [comment]
@tremarkley queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/299).

**Comment by tremarkley** [comment]
@tremarkley merged this pull request with [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/299).

**Pull request merged**